compiler-cli/src/fs.rs

```diff
@@ -41,7 +41,7 @@ impl ProjectIO {
 }
 
 impl FileSystemReader for ProjectIO {
-    fn gleam_source_files(&self, dir: &Path) -> Vec<PathBuf> {
+    fn gleam_ext_files(&self, dir: &Path) -> Vec<PathBuf> {
```

Comment:
We avoid abbreviations, so let's use `extension` rather than `ext` in the method names.